embuilder.py

```diff
@@ -260,14 +267,21 @@ def main():
         clear_port(what)
       if do_build:
         build_port(what)
+    elif what == 'rebuild':
+      if not USE_NINJA:
+        logger.error('"rebuild" target only works with Ninja')
+        return 1
```

Comment:
Its still no clear do me why this isn't its own subcommand.

Reply:
oh oops, yes it is its own top-level operation now (see above); this code is dead and leftover. I'll delete it.